account_asset_management/models/account_asset.py

```diff
@@ -982,7 +987,9 @@ def _compute_depreciation_amount_per_fiscal_year(
         day_amount = 0.0
         if self.days_calc:
             days = (depreciation_stop_date - depreciation_start_date).days + 1
-            day_amount = self.depreciation_base / days
+            purchase_value = self.purchase_value or 0.0
+            salvage_value = self.salvage_value or 0.0
+            day_amount = (purchase_value - salvage_value) / days
```

Comment:
Why would you do that, if depreciation base is precisely purchase_value - salvage_value? https://github.com/OCA/account-financial-tools/blob/14.0/account_asset_management/models/account_asset.py#L306